presto-main/src/main/java/com/facebook/presto/execution/buffer/ClientBuffer.java

```diff
@@ -204,6 +202,46 @@ private synchronized void addPages(Collection<SerializedPageReference> pages)
         }
     }
 
+    /**
+     * Drops pages up to the specified sequence id
+     */
+    public void acknowledgePages(long sequenceId)
```

Comment:
I assume this is a pure move

Reply:
Can we simply make the existing one public, so that it's clear the code was not modified?

Reply:
Yes, actually it is. But need to move the function above because it should collocate with all other public method.